### PR TITLE
Dialect configuration

### PR DESCRIPTION
**Pull request by andrey-yurzanov**
Issue: #27

### CODE REVIEW

File: entity2jooq-codegen-maven-plugin/pom.xml [hunk 1]
```diff
@@ -55,6 +57,19 @@
...
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit-jupiter.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>${junit-jupiter.version}</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (Filkoof) [inline entity2jooq-codegen-maven-plugin/pom.xml:72]: I like when adding comments to "dependency", like  
Whant to offer adding comment in project's poms too

File: entity2jooq-codegen-maven-plugin/src/test/java/io/github/devastool/entity2jooq/codegen/maven/GenerateTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    generate.setDialect(DIALECT);
+    generate.setClassName(CLASS_NAME);
+    generate.setPackageName(PACKAGE_NAME);
+    generate.setTarget(TARGET);
+
+    generate.init(new MavenProject());
+    Assertions.assertEquals(DIALECT, generate.getDialect());
+    Assertions.assertEquals(CLASS_NAME, generate.getClassName());
+    Assertions.assertEquals(PACKAGE_NAME, generate.getPackageName());
+    Assertions.assertEquals(TARGET, generate.getTarget());
```
Comment (Filkoof) [inline entity2jooq-codegen-maven-plugin/src/test/java/io/github/devastool/entity2jooq/codegen/maven/GenerateTest.java:67]: assertAll() here will be great

**Review by Filkoof — COMMENTED** [review]
few not important comments

File: entity2jooq-codegen-maven-plugin/pom.xml (hunk 1 above)
Comment (andrey-yurzanov) [reply entity2jooq-codegen-maven-plugin/pom.xml:72]: Can you give an example of comments to "dependency"?

**Review by Filkoof — APPROVED** [review]

**Review by Sergio-Konovalov — APPROVED** [review]

**Pull request merged**